ngscheckmate/1.3/Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM ubuntu:18.04
+LABEL maintainer="Miguel Brown (brownm28@chop.edu)"
+
+ENV SAMTOOLS_VERSION 1.9
+ENV BCF_VERSION 1.9
```

Comment:
I'm pretty sure this is redundant. Whenever samtools does a release for samtools they do a release for bcftools.